test/harry/main/org/apache/cassandra/harry/util/StringUtils.java

```diff
@@ -22,6 +22,17 @@
 
 public class StringUtils
 {
+    public static String concat(String[] arr)
```

Comment:
isn't this just `java.lang.String#join(java.lang.CharSequence, java.lang.CharSequence...)`?